Self-serve invoice details (#841)
RevenueCat/docs: docs/welcome/set-up-revenuecat/account-management.md

An invoice will be emailed to the owner of a project at the end of the current billing period. If you want to have the invoices emailed to additional email addresses, you can reach out to [Developer Support](https://app.revenuecat.com/settings/support) in order to have them added to your profile for future invoices. You can also view a history of invoices on the [Invoices page](https://app.revenuecat.com/settings/billing/invoices) under the billing category of the project owner's account settings. The history will only list invoices with non-zero billed amounts, so you may see gaps between billing periods if you do not meet the $2.5k limit in every billing period.

### Invoice details

> You can update how your company name, address, and Tax ID/VAT appear on your invoices in [billing settings](https://app.revenuecat.com/settings/billing) under 'Invoice details'. You can also elect to receive invoices at a separate email address (Billing Email) from the email address associated with your RevenueCat account.

Reviewer: [nit]

```suggestion
You can update how your company name, address, and Tax ID/VAT number appear on your invoices in [billing settings](https://app.revenuecat.com/settings/billing) under **Invoice details**. You can also elect to receive invoices at a separate billing email address instead of your RevenueCat account's email address.
```
